### PR TITLE
Filenames for Child processes' logfiles

### PR DESCRIPTION
**Pull request by seongsujeong**
This PR fixes the issue regarding the names of the per-burst process log files when the directories are provided for `--log` argument.

### CODE REVIEW

**Comment by seongsujeong** [comment]
@gshiroma I think this PR is ready to review. As mentioned, I see the few lines of messages as stderr in the per-burst log files (as below).
>[StdErr] /u/aurora-r0/jeong/miniconda3/envs/isce3_dev/lib/python3.11/site-packages/matplotlib/cm.py:478: RuntimeWarning: invalid value encountered in cast
[StdErr]   xx = (xx * 255).astype(np.uint8)

File: src/rtc/rtc_s1.py [hunk 1]
```diff
@@ -74,18 +74,21 @@ def split_runconfig(cfg_in,
...
     for burst_id in list_burst_id:
         path_temp_runconfig = os.path.join(cfg_in.scratch_path,
                                            f'burst_runconfig_{burst_id}.yaml')
-        if path_parent_logfile:
-            path_logfile_child = os.path.join(output_dir_child,
-                                              f'{burst_id}.{path_parent_logfile}')
+        if parent_logfile_path:
+            path_logfile_child = os.path.join(child_output_dir,
+                                              burst_id,
+                                              f'{burst_id}.{basename_logfile}')
```
Comment (gshiroma) [inline src/rtc/rtc_s1.py:90]:
```suggestion
                                              f'{basename_logfile}')
```
Maybe we don't need the f'{burst_id}.` prefix? 

Not for this PR, but I'm thinking now.. another option would be to use the `product_id` here as we do for the other RTC-S1 layers. Then at the end we concatenate all burst logs into the log file that the user set in the CLI command.

**Review by gshiroma — COMMENTED** [review]
Thank you, @seongsujeong . I have two minor comments below. But I think this PR is almost ready for merging.

File: src/rtc/rtc_s1.py [hunk 2]
```diff
@@ -205,7 +208,7 @@ def process_child_runconfig(path_runconfig_burst,
         0 when the child process has completed succesfully
     '''
 
-    os.environ['OMP_NUM_THREADS'] = "1"
+    #os.environ['OMP_NUM_THREADS'] = "1"
```
[the remark this answers is not in the document]
Comment (seongsujeong) [reply src/rtc/rtc_s1.py:211]: Good catch. That was to accelerate the testing. I'll uncomment this.

File: src/rtc/rtc_s1.py (hunk 1 above)
Comment (seongsujeong) [reply src/rtc/rtc_s1.py:90]: I don't have a strong opinion regarding the names for the per-burst log file, so I'll get rid of the prefix.

>Not for this PR, but I'm thinking now.. another option would be to use the product_id here as we do for the other RTC-S1 layers. Then at the end we concatenate all burst logs into the log file that the user set in the CLI command.

I think we can visit this topic after we address other tasks for the delivery?

**Review by gshiroma — APPROVED** [review]
LGTM. Thank you, @seongsujeong

**Pull request merged**